src/xdocs/config_javadoc.xml

```diff
@@ -1628,9 +1628,46 @@ public boolean isSomething()
         </p>
         <source>
 &lt;module name=&quot;SummaryJavadocCheck&quot;&gt;
-    &lt;property name=&quot;period&quot; value=&quot;STRING_LITERAL&quot;/&gt;
+    &lt;property name=&quot;period&quot; value="。"/&gt;
 &lt;/module&gt;
         </source>
+        <p>Code examples:</p>
+
+        <table style="table-layout: fixed;">
+          <tr>
+            <td>
+              Valid Summary java doc
+            </td>
+            <td>
+              Invalid Summary java doc
+            </td>
+          </tr>
+
+          <tr>
+            <td>
+              <source><![CDATA[
+/**
+ * This is valid java doc。
+ */
+public class TestClass {
+    void method() {
+    }
+}
+          ]]></source>
+            </td>
+            <td>
+              <source><![CDATA[
+/**
+ * This is invalid java doc
```

Comment:
please put there simple "." to show clearly that default does not work if user defined smth custom.